components/tutorial/tutorial_tip/tutorial_tip.test.tsx

```diff
@@ -13,6 +13,7 @@ describe('components/tutorial/tutorial_tip/tutorial_tip', () => {
     const currentUserId = 'currentUserId';
     const requiredProps = {
         currentUserId,
+        title: <>{'title'}</>,
```

Comment:
no need to wrap this in a Fragment ... or is it not typed as `ReactComponent`

Reply:
its a ``JSX.Element``

Reply:
well, then the type should be changed, since adding a `Fragment` does seem like overkill, when `ReactNode` (not `ReactComponent`, sry I mixed that up) includes a string.